tests/functional/object/mcg/test_pv_pool.py

```diff
@@ -346,3 +349,36 @@ def test_ephemeral_for_pv_bs(self, backingstore_factory):
         assert (
             "test_object" in search_output
         ), "Dummy data was not found in the node ephemeral storage!"
+
+    @fips_required
+    @tier2
+    @bugzilla("2247731")
+    @polarion_id("OCS-5422")
+    def test_pvpool_bs_in_fips(self, backingstore_factory):
+        """
+        Create PV pool based backingstore and make sure the backingstore doesn't
+        goto Rejected phase on noobaa-core pod restarts
+
+        """
+        # create pv-pool backingstore
+        pv_backingstore = backingstore_factory(
+            "OC",
+            {"pv": [(1, MIN_PV_BACKINGSTORE_SIZE_IN_GB, CEPHBLOCKPOOL_SC)]},
+        )[0]
+
+        # restart noobaa-core pod
+        get_noobaa_core_pod().delete()
+
+        # wait for about 10 mins to check if
+        # the backingstore has reached Rejected state
+        pv_bs_obj = OCP(
+            kind=constants.BACKINGSTORE,
+            namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+            resource_name=pv_backingstore.name,
+        )
+        assert pv_bs_obj.wait_for_resource(
+            condition="Ready", column="PHASE", timeout=600, sleep=5
+        ), "Pv pool backingstore reached rejected phase after noobaa core pod restart"
```

Comment:
@mashetty330 if this test fails, wouldn't it put the test cluster into a bad shape?

Reply:
No it shouldn't, if it does then there is bug 😄